event-hubs.api.md

```diff
@@ -0,0 +1,303 @@
+## API Report File for "@azure/event-hubs"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/abort-controller';
+import { AmqpError } from 'rhea-promise';
+import { AwaitableSender } from 'rhea-promise';
+import { ConnectionContextBase } from '@azure/core-amqp';
+import { DataTransformer } from '@azure/core-amqp';
+import { DefaultDataTransformer } from '@azure/core-amqp';
+import { Dictionary } from 'rhea-promise';
+import { EventHubConnectionConfig } from '@azure/core-amqp';
+import { MessagingError } from '@azure/core-amqp';
+import { Receiver } from 'rhea-promise';
+import { ReceiverOptions } from 'rhea-promise';
+import { RetryOptions } from '@azure/core-amqp';
+import { SharedKeyCredential } from '@azure/core-amqp';
+import { Span } from '@azure/core-tracing';
+import { SpanContext } from '@azure/core-tracing';
+import { SpanOptions } from '@azure/core-tracing';
+import { TokenCredential } from '@azure/core-amqp';
+import { TokenType } from '@azure/core-amqp';
+import { WebSocketImpl } from 'rhea-promise';
+
+// @public
+export interface AbortSignalOptions {
+    abortSignal?: AbortSignalLike;
+}
+
+// @public
+export interface Checkpoint {
+    consumerGroupName: string;
+    eTag: string;
+    eventHubName: string;
+    fullyQualifiedNamespace: string;
+    offset: number;
+    ownerId: string;
+    partitionId: string;
+    sequenceNumber: number;
+}
+
+// @public
+export interface CheckpointManager {
+    listCheckpoints(fullyQualifiedNamespace: string, eventHubName: string, consumerGroup: string): Promise<Checkpoint[]>;
+    updateCheckpoint(checkpoint: Checkpoint): Promise<string>;
```

Comment:
Why is `updateCheckpoint` returning a string?
In the old model this would etag, but after https://gist.github.com/srnagar/a81ec29f4a742952d0875036d1d4732d, we dont need the etag from here anymore right?

Reply:
It's been removed.